setup.py

```diff
@@ -88,7 +88,8 @@ def run_tests(self):
 
 # enum34 is an enum backport for earlier versions of python
 # funcsigs backport required for vendored debtcollector
-install_requires = ["enum34; python_version<'3.4'", "funcsigs>=1.0.0;python_version=='2.7'"]
+# encoding using msgpack
+install_requires = ["enum34; python_version<'3.4'", "funcsigs>=1.0.0;python_version=='2.7'", "msgpack>0.5.6"]
```

Comment:
why `0.5.6`?

Reply:
especially since we have code that supports < 0.4.0 IIUC above